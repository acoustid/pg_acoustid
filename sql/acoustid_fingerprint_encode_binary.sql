SELECT acoustid_fingerprint_encode_binary('{}v1'::acoustid_fingerprint);
SELECT acoustid_fingerprint_encode_binary('{198178303,198047167,164426175}v1'::acoustid_fingerprint);
