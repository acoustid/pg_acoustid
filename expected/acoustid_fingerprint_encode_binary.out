SELECT acoustid_fingerprint_encode_binary('{}v1'::acoustid_fingerprint);
 acoustid_fingerprint_encode_binary 
------------------------------------
 \x01000000
(1 row)

SELECT acoustid_fingerprint_encode_binary('{198178303,198047167,164426175}v1'::acoustid_fingerprint);
    acoustid_fingerprint_encode_binary    
------------------------------------------
 \x01000003499224919224c992083fee03801001
(1 row)

